src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -150,6 +150,36 @@ include::partial$handle-constraints.adoc[]
 
 |===
 
+[#generate-syhtetic-data]
+=== Generate synthetic data for a module
+
+{PRODUCT_NAME} provides a tool to generate synthetic data for a module.
+You can use it to generate a large amount of data for a module to test the application's performance or to demonstrate the implementation.
+
+The tool is currently only available in the command line interface (CLI) and supports synthetic data generation for {APP_NAME_COMPOSE} records (`compose records compose records`) and users (`users synthetic`)
+
+Generated records are flagged with `synthetic:true` (meta), and users will have their handles prefixed with "synthetic_" and email domain "synthetic.tld".
```

Comment:
The `meta` is sort of just thrown in there and it doesn't really matter in this context (IMO); I would phrase it along the lines of

```
Generated records are assigned the `synthetic` flag, and the users have their...
 
[NOTE]
====
Flags are stored in the `meta` attribute ...
====
```

Reply:
Consider doing

```
`synthetic_`
```

instead of `"synthetic_"` -- makes it look more programmer and that's the style we generally use elsewhere